index.js

```diff
@@ -295,6 +297,9 @@ let ContainerService =
     });
 
     button.appendChild(popup);
+    let style = Style({ uri: self.data.url("chrome.css") });
```

Comment:
this method is meat to be called several times for the same window. We should check that this style is not done multiple time. But this can be done in separate patch.
Please add a comment before this let style ...